src/opm/parser/eclipse/EclipseState/SummaryConfig/SummaryConfig.cpp

```diff
@@ -1106,4 +1118,8 @@ bool SummaryConfig::operator==(const Opm::SummaryConfig& data) const {
            this->summary_keywords == data.summary_keywords;
 }
 
+bool SummaryConfig::createRunSummary() const {
+    return this->hasKeyword("RUNSUM");
```

Comment:
In this implementation the `RUNSUM` keyword is treated as "just-another" summary keyword; that is not entirely correct. Instead the `SummaryConfig` class should get a new private member:
```
bool create_run_summary;
```
which should be behind the `createRunSummary()` method.

Reply:
This will be implemented in the morning.